### PR TITLE
[1LP][RFR] Fix polarion docstrings

### PR DESCRIPTION
**Pull request by mshriver**
1. Update polarion-docstrings to include validation that required docstring fields are not set to `None`
2. Correct all currently unassigned cases
3. Default all initialEstimate fields to 1/4h

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 56748c49d08b59b40b72937cf667ccba859ceae8:

`cfme/tests/infrastructure/test_snapshot.py`:
- :bangbang: Line 249:9: **E124** *closing bracket does not match visual indentation*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/cloud/test_providers.py [hunk 1]
```diff
@@ -424,7 +424,7 @@ def test_api_port_max_character_validation_cloud(appliance):
 
 
 @pytest.mark.tier(2)
-@pytest.mark.uncollectif(lambda provider: not provider.one_of(AzureProvider))
+@pytest.mark.provider([AzureProvider], scope="function", override=True)
```
Comment (jawatts) [inline cfme/tests/cloud/test_providers.py]: uncollecting against Azure? Now collecting against it?

File: cfme/tests/containers/test_alerts.py [hunk 2]
```diff
@@ -20,7 +16,7 @@
 def test_add_alerts_provider(provider):
     """
     Polarion:
-        assignee: None
-        initialEstimate: None
+        assignee: jdupuy
```
Comment (jawatts) [inline cfme/tests/containers/test_alerts.py]: This should still be assigned to me

File: cfme/tests/infrastructure/test_snapshot.py [hunk 3]
```diff
@@ -129,7 +126,7 @@ def test_snapshot_crud(small_test_vm, provider):
 
 
 @pytest.mark.rhv3
-@pytest.mark.uncollectif(lambda provider: not provider.one_of(RHEVMProvider))
+@pytest.mark.provider([RHEVMProvider], override=True)
```
Comment (jawatts) [inline cfme/tests/infrastructure/test_snapshot.py]: same question about the uncollect.

File: cfme/tests/cloud/test_providers.py (hunk 1 above)
Comment (mshriver) [reply cfme/tests/cloud/test_providers.py]: uncollecting against _not_ azure, now marking for azure.

File: cfme/tests/infrastructure/test_snapshot.py (hunk 3 above)
Comment (mshriver) [reply cfme/tests/infrastructure/test_snapshot.py]: uncollecting against _not_ rhevm, now marking for rhevm

File: cfme/tests/cloud/test_providers.py (hunk 1 above)
Comment (john-dupuy) [reply cfme/tests/cloud/test_providers.py]: This is uncollecting only if the provider is not one of Azure. I think it is more direct to simply mark the test with the Azure provider

**Pull request merged**